src/uct/api/uct.h

```diff
@@ -1105,6 +1105,8 @@ UCT_INLINE_API ucs_status_t uct_iface_flush(uct_iface_h iface, unsigned flags,
  * @param [in]    flags  Flags that control ordering semantic (currently
  *                        unsupported - set to 0).
  * @return UCS_OK         - Ordering is inserted.
+ *         UCS_ERR_NO_RESOURCE - Ordering cannot be inserted when the routine
+ *                               is called.
```

Comment:
@xinzhao3 it means the flush cannot be **started** because some resources, which typically become available during call to uct_worker_progress, are not available now.